cmd/virt-launcher/node-labeller/node-labeller.sh

```diff
@@ -1,21 +1,24 @@
 #!/bin/bash
 
 set -e
-# if KVM device is not present, exit immediately
-if [ ! -e /dev/kvm ] && [ $(grep '\\<kvm\\>' /proc/misc | wc -l) -eq 0 ]; then
-  exit 0
+
+KVM_MINOR=$(grep -w 'kvm' /proc/misc | cut -f 1 -d' ')
+VIRTTYPE=qemu
+
+if [ ! -e /dev/kvm ] && [ -n "$KVM_MINOR" ]; then
+  mknod /dev/kvm c 10 $KVM_MINOR
 fi
 
-if [ ! -e /dev/kvm ]; then
-  mknod /dev/kvm c 10 $(grep '\\<kvm\\>' /proc/misc | cut -f 1 -d' ')
+if [ -e /dev/kvm ]; then
```

Comment:
Minor nit: I feel like we can combine this into a single conditional for simplicity sake.

Reply:
I'm not sure I see it, could you maybe suggest the specific code change?

Reply:
nvm I misread